### DISCUSSION TITLE
Welcome to geog-312 Discussions!

### DISCUSSION THREAD

**Discussion by giswqs** [giswqs/geog-312#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rayhanulzannat** [comment]
I have anaconda installed in my computer, Should I uninstall anaconda and reinstall miniconda to follow your course?

**Reply by giswqs** [reply]
No. Anaconda is fine

**Reply by richmondkkb** [reply]
This is great. I had the same question.

**Comment by JeanClaude12345** [comment]
Hey all of you!
Am by the name of Jean Claude Twizerimana, from Kigali-Rwanda, am final year student at university of Rwanda!
am apreciative to this georgious course, wishing you all the best!

**Comment by Guy-Marcel** [comment]
Dear professor,
I hope you are well. Thank you for your amazing work on GIS and for sharing all that with us, I appreciate it.
Concerning the certification, I would like to know if we can pay in two parts (i.e $25x25) ?
Regards

**Comment by nallaten** [comment]
I'm confused i need your insight

![Screenshot 2024-09-28 153753](https://github.com/user-attachments/assets/fd0a3436-56ff-41e0-8cba-278499e1c8af)

**Reply by l3oncheung** [reply]
I had the same issue aswell

**Reply by giswqs** [reply]
You need to uncomment the first code block and run it to install the packages.

**Comment by MP-ORTNER** [comment]
Hello Professor, hello to all students,

First I would lie to thank Dr Qiusheng Wu for the amazing work and the sharing of all his knowledge.
I feel very lucky to be able to follow this class and learn new skills.

My name is Marie-Pierre Ortner, I live in Toulouse, France. 
I am graphic designer and I am very found of coding and sciences too. I wish to work as a Data Visualization and Cartography/ GIS digital designer, that is why I have learned QGIS and started to learn Python coding, which led me to this great class.

Best luck to everybody !
